src/sass/footer.scss

```diff
@@ -0,0 +1,54 @@
+@use 'colors.scss';
+.footer {
+  background-color: colors.$blackColor;
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+
+  .footer-nav {
+   
+    display: flex;
+    justify-content: space-around;
+    max-width: 1200px;
+    width: 100%;
+
+    .footer-nav__div {
```

Comment:
Bad practice. Excessive chain of styles in CSS(result example ->.class .class .class { color: 'red'}). Read the documentation.
https://sass-lang.com/documentation/style-rules/